src/test/java/TestUtils.java

```diff
@@ -0,0 +1,19 @@
+import com.itextpdf.text.DocumentException;
+import com.itextpdf.text.pdf.*;
+import com.scrive.pdftools.tools.utils.PdfReaderAutoclosable;
+import com.scrive.pdftools.tools.utils.PdfStamperAutoclosable;
+
+import java.io.*;
+
+public class TestUtils {
+
+    public static boolean checkDocumentIsFlat(byte[] fileFlattened) throws IOException, DocumentException {
+        boolean isflat;
+        try (PdfReaderAutoclosable reader = new PdfReaderAutoclosable(fileFlattened); ByteArrayOutputStream bos = new ByteArrayOutputStream();
```

Comment:
also this  would look nices as 
```suggestion
        try (PdfReaderAutoclosable reader = new PdfReaderAutoclosable(fileFlattened); 
              ByteArrayOutputStream bos = new ByteArrayOutputStream();
```

Reply:
Fixed